_config.yml

```diff
@@ -1,7 +1,5 @@
 # Site settings
 title: SaibroTech
-description: >- # this means to ignore newlines until "baseurl:"
-  site.description
 baseurl: "" # the subpath of your site, e.g. /blog/
 url: "https://www.saibro.tech" # the base hostname & protocol for your site
 author: Copyright &copy; 2021 - SaibroTech
```

Comment:
Aqui de acordo com a [documentação do Jekyll SEO tag](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md) eu posso adicionar as informações da thumbnail do site aqui assim:
```
image:
  path: /assets/img/site-thumbnail.jpg
  height: 627
  width: 1200
```
Não funcionou, então deixei sem

Reply:
Ok, in the next Release we can improve SEO (now our is below 80).